src/Microsoft.DocAsCode.Build.Engine/DocumentBuildParameters.cs

```diff
@@ -31,8 +31,8 @@ public sealed class DocumentBuildParameters : MarshalByRefObject
 
         public TemplateManager TemplateManager { get; set; }
 
-        // todo : partial properties should check.
-        //[IncrementalCheck]
+        // todo : move the check to the prehandle of postbuild phase
+        [IncrementalCheck]
         public ApplyTemplateSettings ApplyTemplateSettings { get; set; }
```

Comment:
`ApplyTemplateSettings` contains absolute path, that's why I remove the attribute `IncrementalCheck`